tests/test_handler_clearml_image.py

```diff
@@ -11,22 +11,38 @@
 
 from __future__ import annotations
 
+import tempfile
 import unittest
+from os import environ
 
 from monai.handlers import ClearMLImageHandler
 from monai.utils import optional_import
 
 Task, has_clearml = optional_import("clearml", name="Task")
+get_active_config_file, has_get_active_config_file = optional_import(
+    "clearml.backend_config.defs", name="get_active_config_file"
+)
 _, has_tb = optional_import("torch.utils.tensorboard", name="SummaryWriter")
 
 
 @unittest.skipUnless(has_clearml, "Requires 'clearml' installation")
 @unittest.skipUnless(has_tb, "Requires SummaryWriter installation")
-@unittest.skip("temp mute clearml tests https://github.com/Project-MONAI/MONAI/issues/6148")
+@unittest.skipIf(not has_get_active_config_file, "ClearML 'get_active_config_file' not found")
+# @unittest.skip("temp mute clearml tests https://github.com/Project-MONAI/MONAI/issues/6148")
```

Comment:
Hi @wyli ,

I think we should remove this comment line? no need to describe the fixed bug in the code.

Thanks.

Reply:
sure I'll remove in https://github.com/Project-MONAI/MONAI/pull/6217